src/MoBi.Presentation/Views/IContainerModalView.cs

```diff
@@ -6,5 +7,7 @@ public interface IContainerModalView : IModalView<IModalPresenter>
    {
       void AddSubView(IView subView);
       bool Show();
+      bool Show(Size modalSize);
```

Comment:
probably size could be optional with a default one? not sure if this is use often